src/server/messages.ts

```diff
@@ -13,7 +13,7 @@ import { parseString } from 'xml2js';
 8. Unpack response to get validation results
 */
 
-const sampleText = `
+/*const sampleText = `
```

Comment:
I think `sampleText` can simply be removed, it's better not to keep old code in comments. @ageorgou thoughts?

Reply:
Yes, now that this is reading the text dynamically, it's not needed (and the tests have their own files). We can and should delete `sampleText`.